modules/@apostrophecms/admin-bar/index.js

```diff
@@ -62,10 +62,10 @@ module.exports = {
       // If `options.contextUtility` is true the item will be displayed in a tray of
       // icons just to the left of the page settings gear. If `options.toggle` is also true,
       // then the button will have the `active` state until toggled
-      // off again. `options.tooltip.active` and `options.tooltip.deactive` may be
-      // provided to offer a different tooltip during the active state. Otherwise
-      // `options.tooltip` is used. The regular label is also present for
-      // screenreaders only. The contextUtility functionality is typically used for
+      // off again. `options.tooltip.deactive` and `options.tooltip.active` may be
```

Comment:
"deactive" should be "inactive"

Also note the stuff that github actions caught.

Reply:
Partial gah. I think my "spell check" did an odd autocorrect of that, correcting it from 'deactivate' to 'deactive'. Corrected.